scripts/gha/build_testapps.py

```diff
@@ -386,9 +374,56 @@ def _validate_android_environment_variables():
           "Neither %s nor %s is set", _ANDROID_SDK_HOME, _ANDROID_HOME)
 
 
+# If sdk_dir is Github repo, then build ios frameworks first
+def _generate_makefiles_from_repo(sdk_dir):
+  sdk_dir = os.path.expanduser(sdk_dir)
+  framework_src_dir = os.path.join(sdk_dir, "frameworks", "ios", "universal")
+
+  # If framework doesn't exist
+  if not os.path.isdir(framework_src_dir):
+    ios_framework_builder = os.path.join(
+      sdk_dir, "build_scripts", "ios", "build.sh")
+
+    framework_builder_args = [
+        ios_framework_builder,
+        "-b", sdk_dir,
+        "-s", sdk_dir,
+        "-c", "false"
+    ]
+    _run(framework_builder_args)
+    return True
+  
+  return False
+
+
+# If sdk_dir is Github repo, then build ios frameworks first
```

Comment:
Is this identical comment supposed to be before both _build_ios_framework_from_repo and _generate_makefiles_from_repo?